pygsti/tools/symplectic.py

```diff
@@ -451,8 +460,10 @@ def compose_cliffords(s1, p1, s2, p2):
         The 'phase vector' over the integers mod 4 representing the compsite Clifford
     """
     assert(_np.shape(s1) == _np.shape(s2)), "Input must be Cliffords acting on the same number of qubits!"
-    assert(check_valid_clifford(s1, p1)), "The first matrix-vector pair is not a valid Clifford!"
-    assert(check_valid_clifford(s2, p2)), "The second matrix-vector pair is not a valid Clifford!"
+    if do_checks:
+        assert(check_valid_clifford(s1, p1)), "The first matrix-vector pair is not a valid Clifford!"
+    if do_checks:
```

Comment:
Could this just be included in the "if" block above?

Reply:
Excellent point... done!